### PR TITLE
add Dockerfile and docker-compose configuration

### PR DESCRIPTION
**Pull request by impca**
fixes bes-dev/stable_diffusion.openvino#13

How to use:
```
docker-compose build
docker-compose run stable diffusion -h
docker-compose run stable-diffusion --prompt "whatever..." --output /tmp/output/space.jpeg
```
Note that models are downloaded and saved to the docker image (Dockerfile, line 16) so the resulting docker image is quite large. 

It would be possible to remove this line and uncomment l. 8 in docker-compose.yaml instead, then models will be downloaded to /tmp/cache on host machine instead.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM debian:bullseye
```
Comment (zackees) [inline Dockerfile:1]: Instead of
FROM debian:bullseye

Consider:
FROM python:3.8.13-bullseye

Then remove the apt-get for python/pip

**Comment by impca** [comment]
Yeah, it needs more changes after the recent commits to master.

File: Dockerfile (hunk 1 above)
Comment (impca) [reply Dockerfile]: Thank you.

**Comment by bes-dev** [comment]
Thanks for your contribution! great stuff! 👍

**Comment by zackees** [comment]
Can we please put this command in the Readme?

```
docker-compose build
docker-compose run stable diffusion -h
docker-compose run stable-diffusion --prompt "whatever..." --output /tmp/output/space.jpeg
```

**Comment by zackees** [comment]
Just fyi this doesn't seem to work on my local windows 10 machine.

```
docker-compose build
docker-compose run stable diffusion -h
--> no such service: stable
```

**Pull request merged**